css/chrome-tabs.css

```diff
@@ -0,0 +1,263 @@
+.chrome-tabs {
```

Comment:
why do we need to include this file? Shouldn't it be already included if you use your forked package 
https://github.com/pedroknup/chrome-tabs/blob/gh-pages/css/chrome-tabs.css

Reply:
Somehow the styles from packages doesn't work on Atom. I think I'd need to somehow import those styles and inject into Atom. I still haven't figured out how to do that.